src/components/Viewport.ts

```diff
@@ -710,7 +710,7 @@ export default class Viewport {
   }
 
   public setCurrentPanel(panel: Panel): void {
-    this.currentPanel = panel;
+    this.currentPanel = panel.getOriginalPanel();
```

Comment:
@WoodNeck What is the purpose of this code? Doesn't it trigger more move?

Reply:
That's my mistake.
Looks like codes from other commit not omitted and included on this one.